README.md

```diff
@@ -1,2 +1,93 @@
 # TV series and movies
- 
+
+JavaScript capstone project - API-based webapp
+
+> This is The JavaScript capstone project about building web application based on the external TVmaze API: data about TV series and movies.API with 2 user interfaces. Done in group of three persons.
+
+## Screenshoots
+
+![TV MAZE](./src/images/tv.png)
+
+### Comment poppup view
+
+![Comments](./src/images/comment.png)
+
+### Reservations poppup view
+
+![Reservations](./src/images/reservation.png)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- Jest
+
+## Live Demo
+
+[live demo](https://tv-maize-zahra-pascal-samiullah-jscapstone.netlify.app/)
+
+## Video Link
+
```

Comment:
Please record a 5 minutes video presenting the features of the project. All the members in the video should appear in the video and talk about the project.

Reply:
Oh my God!  I totally forgot to add the link, thanks for reminding ❤️